contracts/Strategy.sol

```diff
@@ -219,12 +219,11 @@ contract Strategy is BaseStrategy {
         if (_amountNeeded > looseAmount) {
             uint256 toExitAmount = _amountNeeded.sub(looseAmount);
```

Comment:
can change to this, and then remove line 223:
`uint256 toExitAmount = tokensToBpts(_amountNeeded.sub(looseAmount));`

Reply:
We agreed not to introduce this change, right?

Reply:
Correct. the `balanceOfBpt()` check and conditional was determined to be unnecessary because:
- very fringe edgecase
- resolves itself automatically on next harvest or tend